src/equations/shallow_water_multilayer_1d.jl

```diff
@@ -30,6 +30,12 @@ nonconservative term, which has some benefits for the design of well-balanced sc
 The additional quantity ``H_0`` is also available to store a reference value for the total water
 height that is useful to set initial conditions or test the "lake-at-rest" well-balancedness.
 
+Also, there are two thresholds which prevent numerical problems as well as instabilities. Both of 
+them do not have to be passed, as default values are defined within the struct. The limiters are 
+used in [`PositivityPreservingLimiterShallowWater`](@ref) on the water height. `threshold_limiter` 
+acts as a (small) shift on the initial condition and cutoff before the next time step, whereas 
+`threshold_desingularization` is used in the velocity desingularization.
```

Comment:
Possibly here is a better place to include a reference to Chertock regarding the desingularization strategy.